### PR TITLE
Add package: tasknc

### PR DESCRIPTION
**Pull request by matthiasbeyer**
I test-build it on my unstable and I started the application. Works for me, although the package definition does some ugly Makefile modifications.

Also, the software is unmaintained itself. Not sure we want to have this in the repos, but it is a neat toold,... so ... well... you have to decide!

### CODE REVIEW

**Comment by jagajaga** [comment]
Now we name commits another way. Check CONTRIBUTING.md

File: pkgs/applications/misc/tasknc/default.nix [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  name = "tasknc-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/mjheagle8/tasknc/archive/v${version}.tar.gz";
+    sha256 = "0max5schga9hmf3vfqk2ic91dr6raxglyyjcqchzla280kxn5c28";
+  };
+
+  #
+  # I know this is ugly, but the Makefile does serious bullshit in this package,
+  # so we have to:
```
Comment (jagajaga) [inline pkgs/applications/misc/tasknc/default.nix]: I don't like the word `bullshit`.

File: pkgs/applications/misc/tasknc/default.nix [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  #
+  #   1. Remove the "doc" task dependency from the "all" target
+  #   2. Remove the "tasknc.1" task dependency from the "install" target
+  #   3. Remove the installing of the tasknc.1 file from the install target as
+  #      we just removed the build target for it.
+  #
+  # One could also provide a patch for the doc/manual.pod file so it actually
```
Comment (jagajaga) [inline pkgs/applications/misc/tasknc/default.nix]: Write this as todo.

File: pkgs/applications/misc/tasknc/default.nix [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  # me. We have no manpage, though.
+  #
+  preConfigure = ''
+    sed -i -r 's,(all)(.*)doc,\1\2,' Makefile
+    sed -i -r 's,(install)(.*)tasknc\.1,\1\2,' Makefile
+    sed -i -r 's,install\ -D\ -m644\ tasknc\.1\ (.*),,' Makefile
+  '';
+
+  #
+  # I don't know what the author of this software thought when writing the
```
Comment (jagajaga) [inline pkgs/applications/misc/tasknc/default.nix]: Remove this comment.

File: pkgs/applications/misc/tasknc/default.nix (hunk 1 above)
Comment (matthiasbeyer) [inline pkgs/applications/misc/tasknc/default.nix]: will remove it.

Comment (matthiasbeyer) [inline pkgs/applications/misc/tasknc/default.nix]: I guess I also have to apologize for my language. I was a bit... frustrated while writing the package def. I'm sorry.

**Comment by matthiasbeyer** [comment]
I hope I got the commit message accordingly to the new guidelines. I'm not sure whether it should be `tasknc-pkg: init at 0.8` or `tasknc: init at 0.8` - the guide is not clear on this. (I'll submit an issue for this)

**Comment by jagajaga** [comment]
The second is right.

**Comment by matthiasbeyer** [comment]
@jagajaga Then, this should be ready. Thank you for your fast response!

File: pkgs/applications/misc/tasknc/default.nix [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  #
+  # I know this is ugly, but the Makefile does strange things in this package,
+  # so we have to:
+  #
+  #   1. Remove the "doc" task dependency from the "all" target
+  #   2. Remove the "tasknc.1" task dependency from the "install" target
+  #   3. Remove the installing of the tasknc.1 file from the install target as
+  #      we just removed the build target for it.
+  #
+  # @todo One could also provide a patch for the doc/manual.pod file so it
```
Comment (jagajaga) [inline pkgs/applications/misc/tasknc/default.nix]: We usually write `# TODO : foobar`

Comment (matthiasbeyer) [inline pkgs/applications/misc/tasknc/default.nix]: I'm sorry.

**Pull request merged**